virtool/fake/next.py

```diff
@@ -43,6 +44,7 @@
 from virtool.uploads.models import UploadType
 from virtool.uploads.utils import CHUNK_SIZE
 from virtool.users.oas import UpdateUserRequest
+from virtool.subtractions.oas import CreateSubtractionRequest, FinalizeSubtractionRequest, NucleotideComposition
```

Comment:
Please configure Ruff auto-formatter in Pycharm. There is a plugin for it. Use the version and configuration included in this project.